packages/backend-modules/mail/templates/access_recipient_questionnaire_politician.html

```diff
@@ -0,0 +1,125 @@
+                    </p>
+                    <table
+                      width="100%"
+                      border="0"
+                      cellspacing="0"
+                      cellpadding="0"
+                    >
+                      <tr>
+                        <td>
+                          <table border="0" cellspacing="0" cellpadding="0">
+                            <tr>
+                              <td align="center" bgcolor="#3CAD00">
+                                <a
+                                  href="https://tally.so/r/mRd8Bv?uuid={{UUID}}&email={{email}}"
+                                  style="font-size:20px;{{sg_font_style_sans_serif_regular}}color:#ffffff;text-decoration:none;border-radius:0;padding:15px 30px 15px 30px;border:1px solid #3CAD00;display:inline-block;"
+                                  >Zum Fragebogen</a
+                                >
+                              </td>
+                            </tr>
+                          </table>
+                        </td>
+                      </tr>
+                    </table>
+                    
+                    <p>
+                      Sollte dieser Button nicht funktionieren, kopieren Sie bitte den folgenden Link in Ihren Browser: 
+                      <a href="https://tally.so/r/mRd8Bv?uuid={{UUID}}&email={{email}}"
+                        >https://tally.so/r/mRd8Bv?uuid={{UUID}}&email={{email}}</a
+                      >.
+                    </p>
+                    
+                    <p>
+                      Haben Sie noch Fragen? Wir stehen Ihnen gerne unter
+                      <a href="mailto:kontakt@republik.ch"
+                        >kontakt@republik.ch</a
+                      >
+                      zur Verfügung.
+                    </p>
+                    <p>Ihre Crew der Republik</p>
+                  </td>
+                </tr>
+                <tr>
+                  <td style="padding: 20px">
+                    <p
+                      style="color:#282828;font-size:14px;line-height:20px;{{sg_font_style_sans_serif_regular}}"
+                    >
+                      <img
+                        height="79"
+                        src="https://republik.ch/static/logo_republik_newsletter.png"
+                        style="
+                          border: 0px;
+                          width: 180px !important;
+                          height: 79px !important;
+                          margin: 0px;
+                        "
+                        width="180"
+                        alt=""
+                      />
+                      <br />
+                      Republik AG<br />
+                      Sihlhallenstrasse 1, CH-8004 Zürich<br />
+                      <a href="https://republik.ch">www.republik.ch</a><br />
+                      <a href="mailto:kontakt@republik.ch"
```

Comment:
maybe we should use fragebogen-wahlen@republik.ch here too (kontakt@ is the general mail address, I know, but just to be sure, that they use this special email address in case of questions concerning the questionnaire)